README.md

```diff
@@ -73,8 +73,16 @@ Vircadia consists of many projects and codebases with its unifying structure's g
 
 There are many contributors to Vircadia. Code writers, reviewers, testers, documentation writers, modelers, and general supporters of the project are all integral to its development and success towards its goals. Find out how you can [contribute](CONTRIBUTING.md)!
 
-### [Supporters](https://github.com/sponsors/digisomni/)
+### Support
 
-| [Karol Suprynowicz - 74hc595](https://github.com/ksuprynowicz) | 
+You can support the Vircadia project financially through [GitHub Sponsors](https://github.com/sponsors/digisomni/).
+
+You can also support individual active contributors by visiting each repository, this is the main Vircadia repository's [list of active contributors](https://vircadia.com/contributors-redirector/?redirect=vircadia/vircadia). Click on a contributor's profile to see if they accept donations!
```

Comment:
```suggestion
You can support individual active contributors by visiting each repository, this is the main Vircadia repository's [list of active contributors](https://vircadia.com/contributors-redirector/?redirect=vircadia/vircadia). Click on a contributor's profile to see if they accept donations!
```

Reply:
I'm the project's founder so I am fully invested.

I reimburse for hosting fees where possible. I also pay a lot of money for licensing, hosting, domains, and other stuff from day to day, month to month, year to year.

Also I pay people (or are getting into it) to try and get things done in and around the project.

I have and always do reinvest money.